rest_tools/server/config.py

```diff
@@ -58,7 +58,15 @@ def from_environment(keys: KeySpec) -> Dict[str, str]:
     config = keys.copy()
     for key in config:
         if key in os.environ:
-            config[key] = os.environ[key]
+            val = os.environ[key]
+            if config[key] is not None:
+                if isinstance(config[key], bool):
+                    val = True if val.lower() in ('true', 't', '1', 'yes', 'y') else False
```

Comment:
- Do you need safeguards here in case `os.environ[key]` is something unexpected (not `'false', 'f', '0', etc`)?
- this could just be `val = val.lower() in ('true', 't', '1', 'yes', 'y')`, but that's nitpicky

Reply:
I was being lazy and treating everything else as false.  in which case your statement is better